test-suite/shared/src/test/scala/org/scalajs/testsuite/niobuffer/BufferFactory.scala

```diff
@@ -55,8 +59,7 @@ object BufferFactory {
 
     implicit def elemFromInt(value: Int): ElementType = value.toByte
 
-    implicit def elemToJSAny(elem: ElementType): js.Any =
-      js.Any.fromByte(elem)
+    implicit def elemToAnyRef(elem: ElementType): AnyRef = Byte.box(elem)
```

Comment:
I would prefer ascription, mostly because I've always considered `box` methods are more of an implementation detail.